### DISCUSSION TITLE
auto-type curl

### DISCUSSION THREAD

**Discussion by cotangente** [keepassxreboot/keepassxc#6428] [Q&A] [answered]
Hello, I was wondering is there is chance to auto-type when using curl on console/terminal, my exact use-case is "curling" to elasticsearch, for example I can execute:
```
[root@elk ~]# curl -k -X GET -u elastic "https://my-es:9200/_cluster/health?pretty=true"
Enter host password for user 'elastic':
{
  "cluster_name" : "elasticsearch",
...
  "active_shards_percent_as_number" : 100
}
```
...and I have to type my elasticsearch password...

I have 2.6.2, and I'm aware of ssh-agent and it works, but maybe this is slightly different.

cheers.

**Comment by droidmonkey** [comment, marked as answer]
You can use the entry level autotype (Ctrl+Shift+V) or setup a window association to your terminal window. See our user guide for more info.
